packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -88,10 +85,10 @@ @interface FLTCam () <AVCaptureVideoDataOutputSampleBufferDelegate,
 @property(assign, nonatomic) int maxStreamingPendingFramesCount;
 
 @property(assign, nonatomic) UIDeviceOrientation lockedCaptureOrientation;
-@property(assign, nonatomic) CMTime lastVideoSampleTime;
-@property(assign, nonatomic) CMTime lastAudioSampleTime;
-@property(assign, nonatomic) CMTime videoTimeOffset;
-@property(assign, nonatomic) CMTime audioTimeOffset;
+@property(assign, nonatomic) CMTime recordingTimeOffset;
+@property(assign, nonatomic) CMTime lastSampleEndTime;
+@property(nonatomic) AVCaptureOutput *outputForOffsetAdjusting;
+@property(assign, nonatomic) CMTime lastAppendedVideoSampleTime;
```

Comment:
is `lastAppendedVideoSampleTime` the same as `lastSampleEndTime`? 

Can you add some doc on these 4 properties?

Reply:
`lastAppendedVideoSampleTime` is related to this text from the description "Avoid errors when trying to append samples with timestamp less or equal to the previous sample.". `appendPixelBuffer` throws an error if a sample with specified sample time (`PresentationTime`) is appended after another sample with higher or equal time was already appended before (and maybe this should be end time rather than start time too).